microsoft-365/admin/manage/manage-messages.md

```diff
@@ -204,7 +204,7 @@ Need to get a archived message back? No problem.
 
 ## Track your message center tasks in Planner
 
-A lot of actionable information about changes to Microsoft 365 services arrives in the Microsoft 365 message center. It can be difficult to keep track of which changes require tasks to be done, when, and by whom, and to track each task to completion. You also might want to make a note of something and tag it to check on later. You can do all this and more when you sync your messages from the Microsoft 365 admin center to Microsoft Planner. For more information, see [Track your message center tasks in Planner](https://docs.microsoft.comoffice365/planner/track-message-center-tasks-planner).
+A lot of actionable information about changes to Microsoft 365 services arrives in the Microsoft 365 message center. It can be difficult to keep track of which changes require tasks to be done, when, and by whom, and to track each task to completion. You also might want to make a note of something and tag it to check on later. You can do all this and more when you sync your messages from the Microsoft 365 admin center to Microsoft Planner. For more information, see [Track your message center tasks in Planner](https://docs.microsoft.com/en-us/office365/planner/track-message-center-tasks-planner).
```

Comment:
```suggestion
A lot of actionable information about changes to Microsoft 365 services arrives in the Microsoft 365 message center. It can be difficult to keep track of which changes require tasks to be done, when, and by whom, and to track each task to completion. You also might want to make a note of something and tag it to check on later. You can do all this and more when you sync your messages from the Microsoft 365 admin center to Microsoft Planner. For more information, see [Track your message center tasks in Planner](https://docs.microsoft.com/office365/planner/track-message-center-tasks-planner).
```